main.go

```diff
@@ -154,3 +172,41 @@ func printConvertedConfig(opts *options.Options) error {
 
 	return nil
 }
+
+func initTracer(ctx context.Context) (*sdktrace.TracerProvider, error) {
+
+	// Following https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specify-protocol
+	// until https://github.com/open-telemetry/opentelemetry-go/issues/2310 etc is implemented
+	exporterProtocol, exists := os.LookupEnv("OTEL_EXPORTER_OTLP_TRACES_PROTOCOL")
```

Comment:
I'd rather we add tracing configuration to the alpha options structs than have new environments variables configuring this, it doesn't fit with the rest of how this project is configured.

Reply:
This is actually very important when you deploy application in cloud environment. Companies can push this standard variables to the all the apps https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/ .

I would like to say support ENV variables first and have fallback to the alpha or whatever config.